src/TestCaseForTypeCoverage.php

```diff
@@ -110,21 +111,29 @@ public function gatherAnalyserErrors(array $files): array
             foreach ($ruleRegistry->getRules($nodeType) as $rule) {
                 $ruleErrors = $rule->processNode($node, $scope);
                 foreach ($ruleErrors as $ruleError) {
-                    if (is_string($ruleError) || ! $this->ignored($ruleError)) {
-                        $actualErrors[] = $ruleErrorTransformer->transform($ruleError, $scope, $nodeType, $node->getLine());
+                    if ($this->ignored($ruleError)) {
```

Comment:
It looks different than the `2.x` current branch.

Reply:
I look in a minute, but I cloned it yesterday, so I'm wondered, what change since it. I will sync my fork

Reply:
Does it rly? https://github.com/pestphp/pest-plugin-type-coverage/blob/fb76bbac3b4f50f403ec0ee42c9e5301e120bfb6/src/TestCaseForTypeCoverage.php#L113-L115 this is the `2.x` branch, I can't spot a difference, I can't sync my fork, GitHub says, I'm already up-to-date

Reply:
> Does it rly?
> 
> https://github.com/pestphp/pest-plugin-type-coverage/blob/fb76bbac3b4f50f403ec0ee42c9e5301e120bfb6/src/TestCaseForTypeCoverage.php#L113-L115
> 
> this is the `2.x` branch, I can't spot a difference, I can't sync my fork, GitHub says, I'm already up-to-date

If it's in sync, then you've made changes. Revert and keep as original from `2.x` branch: https://github.com/pestphp/pest-plugin-type-coverage/blob/2.x/src/TestCaseForTypeCoverage.php#L93